br/pkg/conn/conn.go

```diff
@@ -299,8 +299,8 @@ func (mgr *Mgr) Close() {
 	mgr.PdController.Close()
 }
 
-// GetTS gets current ts from pd.
-func (mgr *Mgr) GetTS(ctx context.Context) (uint64, error) {
+// GetCurrentTsFromPd gets current ts from pd.
+func (mgr *Mgr) GetCurrentTsFromPd(ctx context.Context) (uint64, error) {
```

Comment:
Maybe rename to `GetCurrentTsFromPD`, as Go usually captialize all letters of an acronym.